timed/employment/serializers.py

```diff
@@ -5,7 +5,8 @@
 from dateutil import rrule
 from django.contrib.auth import get_user_model
 from django.utils.duration import duration_string
-from rest_framework_json_api.relations import ResourceRelatedField
+from rest_framework_json_api.relations import (ResourceRelatedField,  # noqa
+                                               SerializerMethodResourceRelatedField) # noqa
```

Comment:
Instead of the # noqa wouldn't it be better to import relations only?
`from from rest_framework_json_api import relations`